tools/otsdb-dump-restore/README.md

```diff
@@ -60,3 +58,55 @@ Options:
                                 [default: 4096]
   --help                        Show this message and exit.
 ```
+### Format time
```

Comment:
Date and time formats. 
`2023-03-22` is a date

Reply:
done